train.py

```diff
@@ -40,54 +44,51 @@ def main(args):
         train_loss = None
         validation_acc = 0
         validation_loss = None
-        split_val = int(len(dataloader) * 0.8)
-        for batch_num, batch in enumerate(tqdm(dataloader)):
-            if batch_num < split_val:
-                optimizer.zero_grad()
-                
-                img, labels = batch
+        for train_idx, train_batch in enumerate(tqdm(train_dataloader)):
```

Comment:
```suggestion
        for train_idx, train_batch in enumerate(train_dataloader):
```

I don't think tqdm is needed since we print the log out at each 10th iteration while tqdm does that at each iteration, therefore they collide.